notebooks/official/training/hyperparameter_tuning_tensorflow.ipynb

```diff
@@ -954,7 +944,7 @@
         "There are a few arguments to note:\n",
         "\n",
         "\n",
-        "* **max_trial_count**: You need to put an upper bound on the number of trials the service will run. More trials generally leads to better results, but there will be a point of diminishing returns, after which additional trials have little or no effect on the metric you're trying to optimize. It is a best practice to start with a smaller number of trials and get a sense of how impactful your chosen hyperparameters are before scaling up.\n",
+        "* **max_trial_count**: You need to put an upper bound on the number of trials the service runs. More trials generally leads to better results, but there is be a point of diminishing returns, after which additional trials have little or no effect on the metric you're trying to optimize. It is a best practice to start with a smaller number of trials and get a sense of how impactful your chosen hyperparameters are before scaling up.\n",
```

Comment:
change:
- " there is be a point"
to
- there is a point
change:
- "It is a best practice to start with a smaller number of trials.."
- "The best practice is to start with a smaller number of trials..."